includes/manage_pages.inc

```diff
@@ -739,6 +739,10 @@ function islandora_paged_content_zipped_upload_form_submit(array $form, array &$
       'model' => $form_state['values']['model'],
       'file_count' => $file_count,
       'book_label' => $book_label,
+      'ignored_derivs' => array(
+        'ocr' => $form_state['values']['ignore_ocr'],
+        'hocr' => $form_state['values']['ignore_hocr'],
+      ),
```

Comment:
Indeed, good catch! Not following through with the configured form :/ ... think the `isset()` is the cheapest option here, I've updated the pull